sonar-batch/src/main/java/org/sonar/batch/phases/PhaseExecutor.java

```diff
@@ -106,16 +104,11 @@ public void execute(Project module) {
         localIssueTracking();
       }
       issuesReport();
-
-      if (!analysisMode.isPreview()) {
-        persistersExecutor.execute();
-      }
-
       publishReportJob();
       postJobsExecutor.execute(sensorContext);
     }
     cleanMemory();
-    eventBus.fireEvent(new ProjectAnalysisEvent(module, false));
+      eventBus.fireEvent(new ProjectAnalysisEvent(module, false));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)